Dockerfile

```diff
@@ -8,29 +8,10 @@ ADD package.json .
 ADD yarn.lock .
 RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install
 
-
-# Build stage: Run "yarn run build-js"
-# ===
 FROM yarn-dependencies AS build-js
```

Comment:
It's pointless splitting this into 2 stages if the second stage is just going to inherit from the first

Reply:
You also don't need to give your final stage a name. It's not going to be used by anything.

Reply:
Good call, updated.